src/main.rs

```diff
@@ -37,6 +39,14 @@ struct Args {
     /// If not passed, all Wordle games are run.
     #[clap(short, long)]
     games: Option<usize>,
+
+    /// Launch in Interactive mode.
+    ///
+    /// If not passed, the program will run in automatic mode.
```

Comment:
I think it'd be useful to describe briefly what "interactive mode" means here too.

Reply:
Did it !

Reply:
Let's add a little more. Something like:

> This mode is intended for helping you play the game elsewhere. The program will tell you what word to guess next, and ask you for what combination of correct/misplaced/incorrect you got in return.